tests/subscription.test.ts

```diff
@@ -50,10 +52,17 @@ describe("subscriptions.test.ts", () => {
     admin,
     organization,
   }) => {
-    await admin.setSubscriptionState({
-      active: false,
-      ids: [organization.did],
+    const now = new Date();
+    const start = advance(now);
+    const end = advance(start);
```

Comment:
I needed something similar when I was working on maintenance endpoint, I ended up using Temporal. 

```suggestion
    const now = Temporal.Now.instant();
    const start = now.add({ hours: 24 });
    const end = start.add({ hours: 24 });
```